### PR TITLE
v49.14.0 rel notes

### PR DESCRIPTION
**Pull request by jshackell-sfdc**

### CODE REVIEW

File: releasenotes/README.md [hunk 1]
```diff
@@ -10,12 +10,41 @@ If you use [autocomplete](https://developer.salesforce.com/docs/atlas.en-us.sfdx
 
 [Click here for the v48 release notes.](./v48.md)
 
+## 49.14.0 (October 8, 2020)
+
+We aren't releasing a Salesforce CLI patch this week. But we have a few announcements. 
+
+* NEW:  Want to play with the next version of Salesforce CLI before it's live? We target publicly releasing a candidate by Tuesday, although it could be anytime before Thursday, the official release day. Check [open pull requests](https://github.com/forcedotcom/cli/pulls) for early drafts of the release notes. If you find any issues or have feedback after testing, create a Github issue [here](https://github.com/forcedotcom/cli/issues) and include the release candidate version in the title. 
+
+    To install the early release candidate of the CLI to test, for example, that it works correctly with your CI jobs, run this command:
```
Comment (ekapner) [inline releasenotes/README.md]: feels like it's missing a word, like: ...to test, for example, "so" that it works correctly...

File: releasenotes/README.md [hunk 2]
```diff
@@ -10,12 +10,41 @@ If you use [autocomplete](https://developer.salesforce.com/docs/atlas.en-us.sfdx
...
+
+    To install the early release candidate of the `salesforcedx` plug-in to test the `force` commands using your current CLI version, run this command:
+
+     `$ sfdx plugins:install salesforcedx@latest-rc`
+
+    To stop getting early candidates and go back to using the `salesforcedx` plug-in version bundled with the CLI, run this command:
+
+    `$ sfdx plugins:uninstall salesforcedx`
+
+* CHANGE: To improve usability and better align how the Salesforce developer tools have evolved over time, we're reorganizing the Salesforce DX Developer and Setup Guides. In particular, we've moved most of the CLI Configuration and Tips topics to the [Setup Guide](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_intro.htm). Next up are the Dev Hub topics which will move to the [Developer Guide](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_intro.htm). We’ll fix internal cross-references, but you might need to refresh your bookmarks.
```
Comment (ekapner) [inline releasenotes/README.md]: Next up are the Dev Hub topics, (comma) which will move to ...

Comment (ekapner) [reply releasenotes/README.md]: align "with"?

**Review by ekapner — COMMENTED** [review]
Just a couple editorial suggestions

File: releasenotes/README.md (hunk 1 above)
Comment (jshackell-sfdc) [reply releasenotes/README.md]: yep, thx

File: releasenotes/README.md (hunk 2 above)
Comment (jshackell-sfdc) [reply releasenotes/README.md]: si si si.

**Pull request merged**